enterprise/admin/views.py

```diff
@@ -508,16 +509,13 @@ def notify_enrolled_learners(cls, enterprise_customer, request, course_id, users
                 ).format(course_id)
             )
             return
-        course_url = course_details.get('marketing_url')
-        if course_url is None:
-            # If we didn't get a useful path to the course on a marketing site from the catalog API,
-            # then we should build a path to the course on the LMS directly.
-            course_url = get_reversed_url_by_site(
-                request,
-                enterprise_customer.site,
-                'about_course',
-                args=(course_id,),
-            )
+        course_path = urlquote("/courses/{course_id}/info".format(course_id=course_id))
```

Comment:
This needs to be `/courses/{course_id}/course`. I think that changed recently.

Reply:
From what I found Info and Course are two separate tabs. If you go to 'info' it redirects to course if it doesn't exist. Wouldn't we want to drop them into the course description tab if it exists?

Reply:
Maybe check with @andy-armstrong on the Learner team about those courseware URLs. It looks like they are somewhat in flux.

Reply:
Thanks @douglashall. The info page is no longer shown in production, but instead it shows the unified '/course' page. Note that there is a redirect from /info so your code will continue to work.

We have a helper function in the LMS but that won't help you here. I would recommend that you use ``reverse`` rather than hard-coding the URL path. Something like:

    reverse('info', kwargs={'course_id': course_id})

The new course page equivalent would be:

    reverse('openedx.course_experience.course_home', kwargs={'course_id': course_id})

Reply:
Thanks for the info Andy. That explains what I was seeing with the /info and /course behaviour.